_episodes/identifiers.md

```diff
@@ -1,22 +1,85 @@
 
-CMIP5 and CMIP6 (lessons learned; version/concept licence)
+Assigning a PID to your data makes sense. It will make the data easier to find, promotes reproducibility and reuse, makes it possible to track the impact, and enables long-term availability of the data. Remember that we said that data should be *fair* to computers? By assigning PIDs to our datasets (and publications), large networks of linked data and information can be created automatically.
 
-Other (MIP) projects
+> ## Anatomy of a DOI
+> A DOI takes the form of a string with two parts, a prefix and a suffix, divided by a slash. Usually, a prefix corresponds to the publisher, and the suffix to the resource.
+> ![Anatomy of a DOI](../fig/doi_anatomy.png)
+{: .callout}
+
+> ## Exercise
+> - Where does the above linked DOI point to?
+> - Do you think this DOI will still be valid in 50 years?
+> - Who is responsible to update the DOI if you get a dead link?
+> - What do you think should happen if the journal ceases to exist?
+{: .challenge}
+
+Which data need a PID?
+----------------------
+
+In principle, all data (publications, software, observations, model output, figures, etc.) need a PID, but practically, this can be broken down into two types of data:
+- Raw data
+- Processed data
+
+Making raw data available enables other researchers to re-use your data, and to develop their own models and ideas. Making the processed data available serves to promote reproducibility, and large inter-comparison studies.
+
+If you think about it, these concepts are key to the success of large MIP projects in climatology, where many institutes and groups have made their data available to everyone.
+
+One of the challenges is how to deal with 'dynamic data', which is also highly relevant for climatological data. How do you make PIDs for data that is continuously evolving? [For the CMIP6 project](https://datascience.codata.org/articles/10.5334/dsj-2017-030/), a pragmatic approach was taken, consisting of a global PID for the entire, evolving data superset, and specific PIDs for citing specific subsets.
+
+> ## Discussion
+> Consider your own use case:
+> - Does the data have a DOI?
+> - Why/Why not?
+{: .challenge}
+
+How do you get a PID?
+---------------------
+
+Getting a PID (usually a DOI) for your data is as easy as publishing your data to a data archive. After the upload, a PID is automatically assigned. Note that just getting a PID is not all for making the data FAIR. Without metadata it will still be difficult to find the data. That's why many data archives make it easy to metadata, such as a list of authors, related projects, linked publications or other related items.
```

Comment:
```suggestion
Getting a PID (usually a DOI) for your data is as easy as publishing your data to a data archive. After the upload, a PID is automatically assigned. Note that just getting a PID is not all for making the data FAIR. Without metadata it will still be difficult to find the data. That's why many data archives make it easy to add metadata, such as a list of authors, related projects, linked publications or other related items.
```